GeneratedLibraries.proj

```diff
@@ -30,14 +34,22 @@
   <!-- Download discovery documents used as input to code generation. -->
   <Target Name="DownloadDiscoveryDocuments" Condition="!Exists('DiscoveryJson')">
```

Comment:
Since you replaced other hard coded references to `DiscoveryJson` with `$(DiscoveryDocumentsDir)`, maybe also replace this? And there are also two other references to `DiscoveryJson` still in the file.